src/main.cpp

```diff
@@ -187,9 +188,8 @@ static uint64_t get_btime(void)
   std::ifstream file("/proc/stat");
   if (!file)
   {
-    std::cerr << "Fail to open file /proc/stat: " << std::strerror(errno)
-              << std::endl
-              << "Builtin function strftime won't work properly." << std::endl;
+    LOG(ERROR) << "Fail to open file /proc/stat: " << std::strerror(errno)
```

Comment:
```suggestion
    LOG(ERROR) << "Failed to open file /proc/stat: " << std::strerror(errno)
```

Reply:
There are many other occurences of 'Fail to' in current error/warning messages, would you think we should replace them all with 'Failed to'?

Reply:
Let's just change the ones we're touching for now. Not a big deal, just grammar